reference/src/client/include/sa_ta_types.h

```diff
@@ -189,38 +189,41 @@ typedef struct {
 
 typedef struct {
     uint8_t iv[GCM_IV_LENGTH];
-    size_t iv_length;
+    uint64_t iv_length;
     uint8_t tag[MAX_GCM_TAG_LENGTH];
-    uint8_t tag_length;
+    uint64_t tag_length;
```

Comment:
Do we expect to see tag lengths that are 64-bits in size?

Reply:
No, but it was formerly a size_t which is 64 bits on a 64 bit machine and 32 bits on a 32 bit machine. This just standardizes the size for marshalling.

Reply:
It looks like it was previously a uint8_t not a size_t. Did you mean to expand it to 64 bits?

Reply:
Ah it was is a typo here. In sa_types.h, the field is a size_t.